docs/settings/security-settings.asciidoc

```diff
@@ -30,6 +30,112 @@ You do not need to configure any additional settings to use the
 
 |===
 
+[float]
```

Comment:
__issue:__ same comment as in 7.3 version of this PR https://github.com/elastic/kibana/pull/96473#discussion_r609585716. Unfortunately we didn't support this new config format until 7.7. As I mentioned in that comment we introduced a new workaround-y setting in 7.5 that was eventually fully deprecated, so I'm not 100% sure we need to document it. Documenting it is the right thing to do, but I suspect it won't be super useful in practice.